homeassistant/components/smartthings/light.py

```diff
@@ -175,6 +188,18 @@ async def async_set_level(self, brightness: int, transition: int):
         duration = int(transition)
         await self._device.set_level(level, duration, set_status=True)
 
+    @property
+    def color_mode(self) -> ColorMode:
+        """Return the color mode of the light."""
+        if len(self._attr_supported_color_modes) == 1:
+            # The light supports only a single color mode
+            return list(self._attr_supported_color_modes)[0]
+
+        # The light supports hs + color temp, determine which one it is
+        if self._hs_color and self._hs_color[1]:
+            return ColorMode.HS
+        return ColorMode.COLOR_TEMP
```

Comment:
I could not find anything in the `pysmartthings` which helps determine which mode a light is in.